src/NoteMouseHandler/PencilMouseHandler.js

```diff
@@ -80,21 +80,20 @@ export const createNoteAction = ctx => (onMouseDown, onMouseMove) => {
   let startPosition
   let startNote
 
-  onMouseDown(e => {
-    const pos = getLocal(e)
-    startPosition = pos
+  onMouseDown(local => {
+    startPosition = local
 
     const note = createNote(
-      quantizer.floor(transform.getTicks(pos.x)),
-      Math.ceil(transform.getNoteNumber(pos.y)),
+      quantizer.floor(transform.getTicks(local.x)),
+      Math.ceil(transform.getNoteNumber(local.y)),
       quantizer.unit
     )
     track.addEvent(note)
     SharedService.player.playNote(note)
     startNote = _.clone(note)
   })
 
-  onMouseMove(e => { onNoteMouseMove(ctx, e, startPosition, startNote) })
+  onMouseMove(local => { onNoteMouseMove(ctx, local, startPosition, startNote) })
```

Comment:
__[jshint]__
*  'arrow function syntax (=>)' is only available in ES6 (use 'esversion

<!-- Comment by [SideCI](https://sideci.com) -->